### PR TITLE
Just some tidying up of plain English + whitespace + file mode fixes

### PR DESCRIPTION
**Pull request by ZoeB**

### CODE REVIEW

File: src/mame/drivers/alpha68k.c [hunk 1]
```diff
@@ -110,7 +110,6 @@ Stephh's additional notes (based on the games M68000 code and some tests) :
   - There is no Dip Switch to determine if you are allowed to continue a game
     or not, so you ALWAYS have the possibility to continue a game.
 
-
```
Comment (cuavas) [inline src/mame/drivers/alpha68k.c]: Could you please reinstate the extra blank line?  The double blank line separates sections as opposed to the single blank line separating paragraphs in this block comment.

File: src/emu/machine/vrc4373.h [hunk 2]
```diff
@@ -115,12 +115,10 @@ class vrc4373_device : public pci_host_device {
 	UINT32 m_simm_base;
 	dynamic_array<UINT32> m_simm;
 
-
```
Comment (cuavas) [inline src/emu/machine/vrc4373.h]: Please restore this double blank line separating memory configuration from internal registers.

File: src/osd/sdl/README_SDL20.txt [hunk 3]
```diff
@@ -17,15 +17,15 @@ Known bugs:
 * SDL2.0: sdlvideofps does not take -numscreens>1 into account.
 * SDL1.3/WIN32: crashes with -rd d3d
 * SDL1.3/WIN32: resizing does not work
-      
+
 Build SDL 2.0 from HG
 ======================
 
-Pull 2.0 from hg. Than 
+Pull 2.0 from hg. Than
```
Comment (cuavas) [inline src/osd/sdl/README_SDL20.txt]: You might want to correct the then/than confusion as well as removing the trailing whitespace.

**Comment by cuavas** [comment]
It's mostly OK, I've made a few comments on specific changes.  If address them I'll merge it.
Please note that there are often reasons for double blank lines, so please don't just blindly remove them.

**Comment by ZoeB** [comment]
Done.  Thanks for the feedback!  As you might gather, I've only made much smaller C programs and aren't familiar with all these conventions yet.  Thank you!

**Comment by happppp** [comment]
We actually have a tool "srcclean" for whitespace cleanup, scheduled to run
once with every new binary MAME release.

Look the commit message during release that's usually titled something like
'cleanups and version bump' if you want to see what it specifically
changes. Such whitespace cleanup is not needed to be done manually. :)

**Comment by ZoeB** [comment]
Neat!  Checking it out now, thank you.  I'll stick to fixing typos, capitalisations and the like then. :)  Thanks!

**Comment by mr-t-guru** [comment]
Thanks for taking the time to fix those mistakes. 
I wanted to point out another related issue.... since MAME is coded by many people and some of them don't use English as their first language, there are a lot of broken English or grammar errors in the source. So you could make corrections there too if you wanted :-)
I don't want to point fingers but here is one example....
https://github.com/mamedev/mame/blob/master/src/mame/drivers/hng64.c#L631
the comment should be something like "TODO: reads to i/o but coins don't work?"
Or "TODO: reads to i/o but coins are not working?"
Or "TODO: reads to i/o but coin is not working?"
You can check things online using sites like this....
http://www.reverso.net/spell-checker/english-spelling-grammar/
If you check this example there it corrects it to 'don't work'

**Comment by ZoeB** [comment]
Can do!

I wrote a simple program yesterday that only prints out comments, leaving code alone.  I may make a wordlist based on http://en.wikipedia.org/wiki/Commonly_misspelled_English_words and use that with spell, for starters.  Grammatical errors are a little trickier to find (even for that reasonably straightforward example, something like "grep '[^Tt][^h][^i]s does' src/mame/drivers/*.c" is too simplistic), but I'm certainly game to give it a go, sure!

I'm sure I'm not the first woman to like MAME. :P  Centipede was co-created by a woman, Dona Bailey, remember.  It wasn't until 1984 that it was decided that women shouldn't like programming: http://www.npr.org/blogs/money/2014/10/21/357629765/when-women-stopped-coding  There are good efforts underway to get us back on track.

I've also managed to get my nieces into Rod-Land, Uo Poko, etc, so that's neat.  It's a shame MAME doesn't yet support Donkey Kong Pauline Edition, even if it never did make it into an arcade, even as a bootleg.  Little things like that would really help, I'm sure.

**Comment by MooglyGuy** [comment]
There's a number of women who are programmers where I work, and there are a number of women who are active contributors to MAME and MESS. Unfortunately, Pauline Edition will never be added to MAME, because it is a home-made game that was never sold and never actually put on location to make money. It is, however, supported by HBMAME, which is a parallel project that is designed specifically for homemade games.

By the way, I'm not sure where that 1984 figure came from. Maybe Sophie Wilson being the linchpin of the development of the ARM chipset in 1985 and later was a fluke?

**Comment by ZoeB** [comment]
I wasn't aware of HBMAME.  Fantastic, thanks!  I'm glad to hear there are more women contributing to MAME and MESS too, excellent.  Sophie Wilson's great, I went to a talk of hers at Wuthering Bytes last year.  And of course Lynn Conway did some pretty revolutionary work too!

**Comment by MooglyGuy** [comment]
I agree, HBMAME is great! It fills a major niche that isn't necessarily covered by MAME and MESS. MAME and MESS tend to focus on documented commercial releases, but MESS can also be considered HBMESS since the loading of software isn't coupled to definitions in code. By contrast, in MAME there's a 1:1 mapping between a ROM definition in code and and a ROM archive, so HBMAME ensures that people who have taken the time to make modifications to arcade games, or even their own original arcade games, don't see their work lost to the ether.

As an aside, if you'd like any suggestions of other family-friendly puzzle games that your nieces might enjoy, I'm a huge fan of 2D puzzle games, so I can offer plenty of suggestions! You might consider giving Star Sweep a try, there isn't a weekend that goes by when I don't spend an evening trying to get through the whole puzzle progression in one coin. :)

**Comment by ZoeB** [comment]
Thanks for the recommendation!  I'll check it out.  It looks similar to Pnickies, surely a good thing. :)

**Comment by nitrofurano** [comment]
besides hbmame, i think MAME, by default, should be more flexible for running homebrews (for example, via EmulationStation, it's difficult to have PacMan and PacManicMinerMan in the same directory - alternatedly and optionally, a kind of metafile inside the .zip file (saying which machine is to be emulated for opening the rom files) would be great... ) - since months ago, i was getting a lot of fun using Boriel’s ZX-Basic Compiler for making experiences on hardware emulated by MAME http://www.boriel.com/wiki/en/index.php/ZX_BASIC:Released_Programs#Arcade_Systems (only experiences there for now, i really have to code "real" stuff soon! :D ) - and developing homebrews might be an interesting tool for improving MAME emulation when comparing the emulation with the same content burnt into roms and socketed on the real hardware

**Pull request merged**